openassessment/xblock/test/test_staff_area.py

```diff
@@ -23,8 +25,12 @@
 from openassessment.workflow import api as workflow_api
 from openassessment.xblock.data_conversion import prepare_submission_for_serialization
 from openassessment.xblock.test.base import XBlockHandlerTestCase, scenario
+from openassessment.xblock.test.test_team import MockTeamsService, MOCK_TEAM_MEMBERS, MOCK_TEAM_NAME
+from openassessment.xblock.staff_area_mixin import conv_format_list
 from submissions import api as sub_api
 
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
```

Comment:
What is the purpose of this import? Are we logging anything?

Reply:
was.  Have removed in cleanup commit.